src/elements/content-sidebar/activity-feed/annotation-thread/__tests__/AnnotationThreadCreate.test.js

```diff
@@ -0,0 +1,98 @@
+// @flow
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react';
+import { IntlProvider } from 'react-intl';
+import AnnotationThreadCreate from '../AnnotationThreadCreate';
+import { annotation } from '../../../../../__mocks__/annotations';
+
+jest.mock('react-intl', () => jest.requireActual('react-intl'));
+jest.mock('../../comment-form', () => props => {
+    return (
+        <div>
+            {props.isOpen && (
+                <div>
+                    <button type="button" onClick={props.onCancel}>
+                        Cancel
+                    </button>
+                    <button type="button" onClick={() => props.onSubmit('example message')}>
+                        Post
+                    </button>
+                </div>
+            )}
+        </div>
+    );
+});
+
+describe('elements/content-sidebar/activity-feed/annotation-thread/AnnotationThreadCreate', () => {
+    const mockHandleCancel = jest.fn();
+    const mockOnAnnotationCreate = jest.fn();
+    const mockOnError = jest.fn();
+    let mockCreateAnnotation = jest.fn();
+
+    const defaultProps = {
+        api: {
```

Comment:
Can we make this a `getDefaultProps` function so to help guarantee that each test run is reset and not sharing any global object? 

We can also default the callbacks to `jest.fn()` in the above function and allow the test to override it with a particular jest mock in the relevant tests to avoid having shared mocks across test runs. That or we should have a `afterEach` where we reset mocks